cartool/main.m

```diff
@@ -50,22 +51,18 @@ -(CGImageRef)image;
 @interface CUIRenditionKey : NSObject
 @end
 
-@interface CUIThemeFacet : NSObject
-
-+(CUIThemeFacet *)themeWithContentsOfURL:(NSURL *)u error:(NSError **)e;
-
-@end
-
 @interface CUICatalog : NSObject
 
 @property(readonly) bool isVectorBased;
 
+-(id)initWithURL:(id)arg1 error:(id*)arg2;
```

Comment:
This initializer replaces the need to use the `CUIThemeFacet` (in my limited testing)